cli/loader_win_utils.c

```diff
@@ -15,8 +15,8 @@ FILE *stderr = &_stderr;
 int loader_fwrite(const char *str, size_t nchars, FILE *out) {
     DWORD written;
     if (out->isconsole) {
-        // Windows consoles do not support UTF8, only UTF16.
-        size_t wbufsz = nchars * 4 + 1;
+        // Windows consoles do not support UTF-8 (for reading input, though new Windows Terminal does for writing), only UTF-16.
+        size_t wbufsz = ((nchars * 3) >> 1) + 2;
```

Comment:
```suggestion
        size_t wbufsz = nchars * 2 + 2;
```

Reply:
Do you really think it needs 2x not 50%? When?

Reply:
Code points U+0000 to U+007F (ASCII space) require 1 utf8 byte and 2 utf16 bytes. It is the most common case (for filenames).

Reply:
Isn't it always good to allocate least possible? It's only one more assembly instruction.